Rego/SharepointConfig.rego

```diff
@@ -1,265 +1,285 @@
+# MS.SHAREPOINT.3.1v1
 #--
-# At this time we are unable to test for X because of Y
+# At this time we are unable to test for sharing settings of specific SharePoint sites
+# because we have yet to find the setting to check
 tests[{
-    "Requirement" : "Sharing settings for specific SharePoint sites SHOULD align to their sensitivity level",
-    "Control" : "Sharepoint 2.3",
+    "PolicyId" : PolicyId,
     "Criticality" : "Should/Not-Implemented",
     "Commandlet" : [],
     "ActualValue" : [],
-    "ReportDetails" : "Currently cannot be checked automatically. See Sharepoint Secure Configuration Baseline policy 2.3 for instructions on manual check",
+    "ReportDetails" : NotCheckedDetails(PolicyId),
     "RequirementMet" : false
 }] {
+    PolicyId := "MS.SHAREPOINT.3.1v1"
     true
 }
 #--
 
-
-################
-# Baseline 2.4 #
-################
+###################
+# MS.SHAREPOINT.4 #
+###################
 
 #
-# Baseline 2.4: Policy 1
+# MS.SHAREPOINT.4.1v1
 #--
-ReportDetails2_4_1(Policy) = Description if {
+ExpirationTimersGuestAccess(Policy) = [ErrMsg, Status] if {
     Policy.SharingCapability == 0
-	Description := "Requirement met"
+	ErrMsg := ""
+    Status := true
 }
 
-ReportDetails2_4_1(Policy) = Description if {
+ExpirationTimersGuestAccess(Policy) = [ErrMsg, Status] if {
     Policy.SharingCapability != 0
     Policy.ExternalUserExpirationRequired == true
-    Policy.ExternalUserExpireInDays == 30
-	Description := "Requirement met"
+    Policy.ExternalUserExpireInDays <= 30
+	ErrMsg := ""
+    Status := true
 }
 
-ReportDetails2_4_1(Policy) = Description if {
+ExpirationTimersGuestAccess(Policy) = [ErrMsg, Status] if {
     Policy.SharingCapability != 0
     Policy.ExternalUserExpirationRequired == false
-    Policy.ExternalUserExpireInDays == 30
-	Description := "Requirement not met: Expiration timer for 'Guest access to a site or OneDrive' NOT enabled"
+    Policy.ExternalUserExpireInDays <= 30
+	ErrMsg := "Requirement not met: Expiration timer for 'Guest access to a site or OneDrive' NOT enabled"
+    Status := false
 }
 
-ReportDetails2_4_1(Policy) = Description if {
+ExpirationTimersGuestAccess(Policy) = [ErrMsg, Status] if {
     Policy.SharingCapability != 0
     Policy.ExternalUserExpirationRequired == true
-    Policy.ExternalUserExpireInDays != 30
-	Description := "Requirement not met: Expiration timer for 'Guest access to a site or OneDrive' NOT set to 30 days"
+    Policy.ExternalUserExpireInDays > 30
+	ErrMsg := "Requirement not met: Expiration timer for 'Guest access to a site or OneDrive' NOT set to 30 days or less"
+    Status := false
 }
 
-ReportDetails2_4_1(Policy) = Description if {
+ExpirationTimersGuestAccess(Policy) = [ErrMsg, Status] if {
     Policy.SharingCapability != 0
     Policy.ExternalUserExpirationRequired == false
-    Policy.ExternalUserExpireInDays != 30
-	Description := "Requirement not met"
+    Policy.ExternalUserExpireInDays > 30
+	ErrMsg := "Requirement not met"
```

Comment:
Why are all of these indented?  Makes it look like it is intended to be an 'if' statement.

Reply:
A copy & paste error, fixed. Please close upon confirming